### DISCUSSION TITLE
If the full game is a modification of the free game, will it be open source?

### DISCUSSION THREAD

**Discussion by P-Star7** [FunkinCrew/Funkin#1867] [Q&A] [answered]
The rules of the license say that if anyone modifies the code in this repository, that they have to open-source it as well. I presume that this applies to the modifications that the developers are making to their code to turn it into the full game. Will this mean that FNF will become one of the first open-source games that can still sell copies and make money?

**Comment by steviegt6** [comment, marked as answer]
Ninja has already said that the full game will be open-source.
